### PR TITLE
Features/popular person

### PR DESCRIPTION
**Pull request by RobFyd**
PersonalDeatailTile styled and responsiveness added. Please check and comment.

### CODE REVIEW

**Review by P-Michalski — CHANGES_REQUESTED** [review]
I think for now, there is so little to be merged yet. Also, on mobile display, the tile should in my opinion be stretched on entire width.

**Comment by RobFyd** [comment]
The tile extends across the entire mobile view. Additionally, I added links from the API. I think the work done is ready to be added. Have fun with merging ;)

File: src/common/PersonDetailTile/index.js [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+    <Wrapper>
+      <DetailsTile>
+        <NoImage
+          src={poster ? `${imagesAPIw400}${poster}` : anonymous}
+          alt="name"
+        />
+        <Content>
+          <Title>{name}</Title>
+          <DetailsLine>
+            <Label>Date of birth:</Label>
```
Comment (waltad) [inline src/common/PersonDetailTile/index.js]: In the mobile version it should only be visible "Brith:"

File: src/common/PersonDetailTile/index.js [hunk 2]
```diff
@@ -0,0 +1,46 @@
...
+            <Info>{birthday}</Info>
+          </DetailsLine>
+          <DetailsLine>
+            <Label>Place of birth:</Label>
+            <Info>{birthplace}</Info>
```
Comment (waltad) [inline src/common/PersonDetailTile/index.js:39]: In the mobile version description ("Place of birth:") should be under brithplace.

File: src/common/PersonDetailTile/index.js [hunk 3]
```diff
@@ -0,0 +1,46 @@
...
+      <DetailsTile>
+        <NoImage
+          src={poster ? `${imagesAPIw400}${poster}` : anonymous}
+          alt="name"
+        />
+        <Content>
+          <Title>{name}</Title>
+          <DetailsLine>
+            <Label>Date of birth:</Label>
+            <Info>{birthday}</Info>
```
Comment (waltad) [inline src/common/PersonDetailTile/index.js]: the date format needs to be changed: DD.MM.RRRR

File: src/apiURL.js
Comment (waltad) [inline src/apiURL.js:1]: Good job. Apart from some minor styling tweaks, everything works fine.

**Review by P-Michalski — APPROVED** [review]

**Comment by P-Michalski** [comment]
Cool, now we can merge 😄

**Pull request merged**